tuxemon/save.py

```diff
@@ -62,6 +63,14 @@
 config = prepare.CONFIG
 
 
+class SaveData(NPCState):
+    screenshot: str  # screenshot encoded as a string
```

Comment:
Maybe we should use a different type to prevent mistakes?

Reply:
What do you mean by this?

Reply:
Using `NewType` for it, to prevent mistakenly assigning a normal string.